src/features/PesterTests.ts

```diff
@@ -54,11 +54,12 @@ export class PesterTestsFeature implements IFeature {
         this.launch(launchConfig);
     }
 
-    private async launchTests(uriString: string, runInDebugger: boolean, describeBlockName?: string) {
+    private async launchTests(uriString: string, runInDebugger: boolean, describeBlockName?: string,
+                              describeBlockLineNumber?: number) {
         // PSES passes null for the describeBlockName to signal that it can't evaluate the TestName.
-        if (!describeBlockName) {
+        if (!describeBlockName && !describeBlockLineNumber) {
             const answer = await vscode.window.showErrorMessage(
-                "This Describe block's TestName parameter cannot be evaluated. " +
+                "This Describe block's TestName parameter cannot be evaluated in versions of Pester before 4.6.0. " +
```

Comment:
Maybe something like this is a little more actionable?

```suggestion
                "This Describe block's TestName parameter cannot be evaluated. Either try again with Pester 4.6.0 or higher, or remove any inline parameters in your TestName." +
```

Thoughts @rkeithhill?

Reply:
Yes, something like this but it might ne even better if the ending is more like `or remove any variable expressions in the Describe block name`